mitodl/micromasters
# Added custom 404 error page

#### What are the relevant tickets?
closes #440

#### What's this PR do?
Adds a custom 404 page

#### Where should the reviewer start?
probably  `ui.views` and then '404.html'

#### How should this be manually tested?
to see the the 404 template, just go to `/404/`.
To actually test that a 404 is returned for a not existing URL, you need to:
  - set the debug to False in your `docker-compose.yml`
  - run `node node_modules/webpack/bin/webpack.js --config webpack.config.prod.js --bail` to build the static assets
  - run `docker-compose run web ./manage.py collectstatic` 

#### Any background context you want to provide?
This story does not address the case the page is not found in the react router

#### What GIF best describes this PR or how it makes you feel?
![](http://replygif.net/i/1353.gif)
